### DISCUSSION TITLE
About console.*

### DISCUSSION THREAD

**Discussion by bestpika** [alagrede/znote-app#64] [Q&A] [answered]
Hello, I sometimes copy and paste code from some of the StackOverflow answers to take notes.
But the code people wrote there doesn't use `print()` to print the result, but `console.*()` to print the result, I don't know if you can make the result printed by `console.*()` to be shown in the executed result too?
Thanks.

**Comment by alagrede** [comment, marked as answer]
Hi @bestpika,
Yes, I know it could be more convenient to have this `console.log` but currently I can only add the minimum `console.log()` for compatibility. The console object does a lot of things that I can't reproduce in the HTML output.
**I'll add it for the next minor version.**

Anthony

**Reply by alagrede** [reply]
You can try the **console.log** in the version 1.7.7

**Reply by bestpika** [reply]
Although the function is still a bit different from the original one, at least it can be displayed now.
Thanks!

**Reply by alagrede** [reply]
Yes, as explained, the `print` method is not a `console.log`. I added minimal compatibility to help you view the results.
